lattice/schema_to_json.py

```diff
@@ -265,8 +266,8 @@ def _construct_requirement_if_then(
         :param requirement:             This item's presence is dependent on the above condition
         """
         separator = r"\sand\s"
-        collector = "allOf"
-        selector_dict: dict = {"properties": {collector: {}}}
+        # collector = "allOf"
```

Comment:
Should this just be removed? If not, it probably needs more context in comments.

Reply:
Very good point; I'll sort it one way or the other in my next brach.